third_party/skia/src/sksl/SkSLPool.cpp

```diff
@@ -32,13 +65,17 @@ Pool::~Pool() {
     fMemPool->reportLeaks();
     SkASSERT(fMemPool->isEmpty());
 
+#ifndef STARBOARD
     VLOG("DELETE Pool:0x%016llX\n", (uint64_t)fMemPool.get());
```

Comment:
No need to necessarily do it in this PR, but we should try and provide a VLOG macro, even as a stub, for STARBOARD once, so we don't have to #ifdef it everywhere.

something like
```
#ifdef STARBOARD
#define VLOG(...) {} 
#endif
```

Not sure which header should it go in.

Reply:
Will fix this in a separate PR to avoid make this code change messier.